test/e2e/singlecluster/pod_test.go

```diff
@@ -96,6 +96,10 @@ var _ = ginkgo.Describe("Pod groups", func() {
 				Image("gcr.io/k8s-staging-perf-tests/sleep:v0.1.0", []string{"1ms"}).
 				Queue(lq.Name).
 				Request(corev1.ResourceCPU, "1").
+				// Adding a resource that is *not* in Cluster Queue but is marked
+				// as excluded in the configuration. Testing that it passes without error.
+				Request("networking.networks/vpc1", "1").
```

Comment:
This Pod will pass Kueue admission, but it will not run, because this is an E2E test and kube-scheduler wouldn't schedule it.

Better test at integration level.

Reply:
Oh, thank you for this -- I could not figure out why e2e was failing. 

Added integration testing.